openstack/utils/templates/_ini_sections.tpl

```diff
@@ -70,7 +70,7 @@ driver = messagingv2
                     {{- include "ini_sections._transport_url" (tuple . $data) }}
                 {{- end }}
             {{- end }}
-mem_queue_size = {{ .Values.audit.mem_queue_size }}
+mem_queue_size = {{ .Values.audit.mem_queue_size | default 1000 }}
```

Comment:
please add `| int }}` to ensure that Helm is really using that as an integer and not as something else...